### PR TITLE
Updated expo-drag-drop-content-view data

### PR DESCRIPTION
**Pull request by AlirezaHadjar**
# 📝 Why & how

# ✅ Checklist

- [ ] Added library to **`react-native-libraries.json`**
- [x] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

- Changed example and gif paths due to migrating to monorepo

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -9881,10 +9881,12 @@
   },
   {
     "githubUrl": "https://github.com/AlirezaHadjar/expo-drag-drop-content-view",
```
Comment (Simek) [inline react-native-libraries.json:9883]:
```suggestion
    "githubUrl": "https://github.com/AlirezaHadjar/expo-drag-drop-content-view/tree/main/packages/expo-drag-drop-content-view",
```

The package URL also need to be updated, to correctly extract the package data after monorepo reorganization.

**Comment by Simek** [comment]
Not strictly related to the changes in this PR, looks like you are using the links in package Readme which leads to Expo API reference, thus they results in 404.

File: react-native-libraries.json (hunk 1 above)
Comment (AlirezaHadjar) [reply react-native-libraries.json:9883]: Correct, thanks for the tip🙂

**Comment by AlirezaHadjar** [comment]
> Not strictly related to the changes in this PR, looks like you are using the links in package Readme which leads to Expo API reference, thus they results in 404.
> 
> Screenshot 2024-03-18 105145

Thanks for the heads up. I just fixed the broken links as well.

**Review by Simek — APPROVED** [review]
Thanks for the updates! 👍

**Pull request merged**